### ISSUE TITLE
Error code on decompress: -2

### ISSUE THREAD

**Issue by RazrFalcon** [shssoichiro/oxipng#54]
Application returns an error on any png file I'm trying to optimize.

```
% oxipng color-prop-02-f.png
Processing: color-prop-02-f.png
Error code on decompress: -2
```

```
% oxipng -V
oxipng 0.14.0
```

**Comment by shssoichiro** [comment]
Can you upload one of the files to this issue so that I can test it on my end?

**Comment by RazrFalcon** [comment]
Any png file. Maybe bug in libraries. I'm using Gentoo, by the way.

I'll try to investigate it.

**Comment by RazrFalcon** [comment]
Tests are failed too:

```
test result: FAILED. 0 passed; 66 failed; 0 ignored; 0 measured
```

With the same error.

**Comment by RazrFalcon** [comment]
Everything works fine on Ubuntu 16.04 with zlib 1.2.8, but on Gentoo with zlib 1.2.11 - not.

The error itself is occurred here: https://github.com/shssoichiro/oxipng/blob/master/src/deflate/libz_stream.rs#L84

**Comment by CounterPillow** [comment]
Can reproduce with zlib 1.2.11 on Arch Linux.

**Comment by shssoichiro** [comment]
It looks like this is also happening on my Gentoo box with zlib 1.2.11, although not on my OS X box (with unknown zlib version). I'll investigate further.

**Comment by shssoichiro** [comment]
Published 0.14.1 which should fix this issue (tested it on both of my machines). It works around the issue by no longer using zlib, and using miniz instead.
